pages/_/profile.js

```diff
@@ -29,14 +32,14 @@ export default class ProfilePage extends React.Component {
     const title = this.props.creator ? `${this.props.creator.username}` : "404";
     const url = `https://slate.host/${title}`;
     const description = this.props.creator.data.body;
+    const image = this.props.creator.data.photo;
+
+    if (!image) {
```

Comment:
if (Strings.isEmpty(image))

Reply:
should I change the `!image` in slate.js to `(Strings.isEmpty(image))` as well?

<img width="998" alt="Screen Shot 2020-11-09 at 3 41 26 PM" src="https://user-images.githubusercontent.com/35607644/98609467-2a05c680-22a2-11eb-8e83-ba4a1d6879de.png">

Reply:
yep!